packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -24,6 +24,7 @@ Requires: katello-selinux
 Requires: foreman-postgresql
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
+Requires: foreman < 2.2
```

Comment:
Could be good to follow the same pattern for foreman_max_version

Reply:
That, or perhaps you can do some math so it becomes `%{foreman_min_version} + 0.1`. Not sure if RPM specs allow this.